src/main/java/org/geysermc/discordbot/commands/IssueCommand.java

```diff
@@ -63,8 +63,8 @@ public IssueCommand() {
         this.guildOnly = false;
 
         this.options = Arrays.asList(
-            new OptionData(OptionType.INTEGER, "number", "The issue/pr number").setRequired(true),
-            new OptionData(OptionType.STRING, "repo", "The repository to lookup, defaults to GeyserMC/Geyser")
+                new OptionData(OptionType.INTEGER, "number", "The issue/pr number").setRequired(true),
```

Comment:
Most of us have `Continuation indent` set to 4 instead of 8 to avoid this. Would recommend setting this as well:
`IntelliJ > Preferences > Editor > Code Style > Java`

Reply:
Ah good to know thanks! 😊